package.json

```diff
@@ -10,7 +10,7 @@
     "verify:clickup": "ts-node src/verify clickup",
     "verify:jira": "ts-node src/verify jira",
     "verify:teamwork": "ts-node src/verify teamwork",
-    "test": "jest",
+    "test": "jest --coverage",
```

Comment:
let's have
"test": "jest"
"test:coverage": "jest --coverage",

and use test:coverage in the pipeline